mco_ovs_supervisor.yml.tmpl

```diff
@@ -12,18 +12,19 @@ spec:
       units:
         - contents: |
             [Unit]
+            Before=kubelet.service
             After=NetworkManager.service openvswitch.service
             [Service]
             Type=oneshot
-            ExecStart=/bin/sh /var/setup-ovs.sh
+            ExecStart=/bin/sh /var/init-interfaces.sh
```

Comment:
I understand that the old name is not fitting anymore, but keeping it has a major benefit - it will make upgrade from one to another much easier.

If we don't keep the name, old MCO will be still available on the cluster, will be applied to all nodes (even new ones) and we will conflict with it.

If we keep the name, MCO will roll-out the upgrade for us: All nodes, one by one will be taken down by MCO,  their original day-1 script will be updated and applied on the system 

I am open to a different cluster upgrade plan. Maybe we can use the same MC to remove old file and install a new one (with nicer name).

Note that even though we assume we will be starting on fresh nodes (without old NM profiles), we don't assume we will be starting on a fresh cluster (so we will have the old MCO to deal with).

Reply:
I guess it will

Reply:
I've kept the mcp name `00-configure-ovs-master/workers`
and also disabled `setup-ovs.service` just in case

Reply:
I think nothing will happen but we need to check it with @kobi86 once this PR is ready